lib/src/hooks/useDynamicForm.tsx

```diff
@@ -254,11 +264,57 @@ export function useDynamicForm(
         },
         [state]
     );
+
+    const meta = React.useMemo(() => {
+        let newMeta: DynamicFormMetaData = {};
+
+        sections?.forEach(({ key: sectionKey, items }) => {
+            newMeta[sectionKey] = {};
+            items?.forEach(({ key, controlType }) => {
+                const itemState: DynamicFormInternalStateValue = state && state[sectionKey] && state[sectionKey][key];
```

Comment:
```
const itemState: DynamicFormInternalStateValue = state && state[sectionKey] && state[sectionKey][key];
```
from the usage, itemState does not seemed to be intended as boolean value, perhaps the intention was to use `||`?

Reply:
Yeah the intention here is to have a falsy value if the value does not exist and the actual value if it does.